helpers/metadata.ts

```diff
@@ -73,7 +73,7 @@ function compileSyncTable(syncTable: GenericSyncTable): PackSyncTable {
   }
 
   const {getter, ...rest} = syncTable;
-  const {execute, ...getterRest} = getter;
+  const {execute, executeUpdate, ...getterRest} = getter;
```

Comment:
So here we're losing any notion in the metadata that this table supports updates. So we'll need to add a new bit to the compiled metadata for `supportsUpdates?: boolean;` or something and set it when `executeUpdate` exists.

Right now `getter` is typed as `PackFormulaMetadata` so we'll need some new sync-specific metadata type.

Reply:
I added something, but it feels a bit clunky.  Let me know if you have a better pattern for this.

Reply:
> Probably not enough to go looking for the actual function in a bunch of places where we want to expose this in the UX.

Yeah, you literally can't find it in the UI (or really anywhere outside of lambda) otherwise, since the existence of the functions are stripped out entirely.

Reply:
Hmm what seems clunky about it? This seems direct and reasonable.